pkgs/os-specific/linux/nix-ld/default.nix

```diff
@@ -1,49 +1,61 @@
-{ stdenv, meson, ninja, lib, nixosTests, fetchFromGitHub }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, meson
+, ninja
+, nixosTests
+}:
 let
-  self = stdenv.mkDerivation {
-    name = "nix-ld";
-    src = fetchFromGitHub {
-      owner = "Mic92";
-      repo = "nix-ld";
-      rev = "1.0.0";
-      sha256 = "sha256-QYPg8wPpq7q5Xd1jW17Lh36iKFSsVkN/gWYoQRv2XoU=";
-    };
-
-    doCheck = true;
-
-    nativeBuildInputs = [ meson ninja ];
-
-    mesonFlags = [
-      "-Dnix-system=${stdenv.system}"
-    ];
-
-    hardeningDisable = [
-      "stackprotector"
-    ];
-
-    postInstall = ''
-      mkdir -p $out/nix-support
-      basename $(< ${stdenv.cc}/nix-support/dynamic-linker) > $out/nix-support/ld-name
-    '';
-
-    passthru.tests.nix-ld = nixosTests.nix-ld;
-    passthru.ldPath = let
-      libDir = if stdenv.system == "x86_64-linux" ||
-                  stdenv.system == "mips64-linux" ||
-                  stdenv.system == "powerpc64le-linux"
-               then
-                 "/lib64"
-               else
-                 "/lib";
-      ldName = lib.fileContents "${self}/nix-support/ld-name";
-    in "${libDir}/${ldName}";
-
-    meta = with lib; {
-      description = "Run unpatched dynamic binaries on NixOS";
-      homepage = "https://github.com/Mic92/nix-ld";
-      license = licenses.mit;
-      maintainers = with maintainers; [ mic92 ];
-      platforms = platforms.linux;
-    };
+  libDir =
+    if
+      stdenv.system
+      == "x86_64-linux"
+      || stdenv.system == "mips64-linux"
+      || stdenv.system == "powerpc64le-linux"
+    then "/lib64"
```

Comment:
WDYT to instead do?

```suggestion
    if builtins.elem stdenv.system [ "x86_64-linux" "mips64-linux" "[powerpc64le-linux" ]
    then "/lib64"
```

Reply:
Also, any reason this affects those specific platforms (e.g.: why it doesn't affect `aarch64-linux` :thinking:?).

Reply:
It's not defined by glibc this way for aarch64. My guess is that aarch64 does not really have multi-lib support.